actors/common/src/main/java/org/sunbird/learner/actors/OrganisationManagementActor.java

```diff
@@ -470,6 +476,28 @@ private void createOrg(Request actorMessage) {
     }
   }
 
+  private void createOrgExternalIdRecord(String channel, String externalId, String orgId) {
+    Map<String, Object> orgExtIdRequest = new HashMap<String, Object>();
+    orgExtIdRequest.put(JsonKey.PROVIDER, channel);
+    orgExtIdRequest.put(JsonKey.EXTERNAL_ID, externalId);
+    orgExtIdRequest.put(JsonKey.ORG_ID, orgId);
+
+    Util.DbInfo orgExtIdInfo = Util.dbInfoMap.get(JsonKey.ORG_EXT_ID_DB);
```

Comment:
Let us not use Util.dbInfoMap anymore. It would unnecessarily hold up memory. Just reuse the constants in the cassandra APIs.